### ISSUE TITLE
Warning color not accessible on white background

### ISSUE THREAD

**Issue by Spark450** [GovAlta/ui-components-design#42]
Figma file went through with Ambrose, front end designer from housing:

https://www.figma.com/file/5TmSNRx5kKkLwAA8GRfQTY/HDS-1461---Housing-Needs-Assessment-Submission?node-id=4832%3A123209 

When using the status-warning colour on a white background (eg. for an icon), it is not meeting accessibility standards.

The warning-dark colour also does not meet accessibility, but get’s very close ([minimum 3:1 contrast ratio](https://www.w3.org/WAI/WCAG21/Techniques/general/G207)).

Need guidance on using the warning colour on white background, and possibly adjust of the darker colour to meet accessibility standards as a way forward.

What implication does this have on other areas of the design system where this colour is used? eg. Warning Badge

**Comment by Spark450** [comment]
[Jira issue](https://goa-dio.atlassian.net/browse/DDIDS-558?atlOrigin=eyJpIjoiZTYxY2I1YjhmMWNmNDcxYjlmN2IyMDk3YTBjZjMyMmEiLCJwIjoiaiJ9)

**Comment by Spark450** [comment]
We will address this as part of this other issue.
